lib/kitchen/directions/bake_page_title.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    module BakePageTitle
+      def self.v1(page:, number:)
+        page.search("div[data-type='description']").each(&:trash)
+        page.add_class('chapter-content-module')
```

Comment:
I don't think these things are really related to "baking a page title"... maybe I suggested calling this the wrong name.  Maybe it is `BakeNonIntroductionPage` or something